sniproxy.cpp

```diff
@@ -157,23 +157,17 @@ SNIProxy::SNIProxy(xcb_window_t wid, QObject* parent):
                              windowMoveConfigVals);
 
 
-    //if the window is a clearly stupid size resize to be something sensible
-    //this is needed as chormium and such when resized just fill the icon with transparent space and only draw in the middle
-    //however spotify does need this as by default the window size is 900px wide.
-    //use an artbitrary heuristic to make sure icons are always sensible
-    if (clientGeom->width < 12 || clientGeom->width > s_embedSize ||
```

Comment:
I don't like this change, they get all different sizes.

I appreciate it solves the scaling problem, but then we're just in "which problem is more annoying" which is personal preference.

Could you mabye try this patch on plasma-frameworks: https://paste.kde.org/p1adgfb1h
it might make the upscaling better; it scales in QPixmap space rather than in OpenGL on the fly 

(though both should be bilinear scaling...so probably not much different)